component/answer/src/main/java/org/exoplatform/management/answer/operations/AnswerExportTask.java

```diff
@@ -35,15 +39,20 @@
  */
 public class AnswerExportTask implements ExportTask {
   public static final String FILENAME = "/category.xml";
+  final private static Logger log = LoggerFactory.getLogger(AnswerExportTask.class);
 
   private final String type;
   private final Category category;
   private final List<Question> questions;
+  private final FAQService faqService;
```

Comment:
Do not use faqService as a variable in the task, ask the container to send it when needed !